Healthy/Classes/Generated/UIColors+Generated.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit.UIColor
+
+// this is automatic generated file please don't edit it 🗡️
+// MARK: - Colors
+
+// swiftlint:disable force_unwrapping
+extension UIColor {
+
+      static var black: UIColor {
+        UIColor(named: "Black")!
+      }
+
+      static var gray1: UIColor {
+        UIColor(named: "Gray 1")!
+      }
+
+      static var gray2: UIColor {
+        UIColor(named: "Gray 2")!
+      }
+
+      static var gray3: UIColor {
+        UIColor(named: "Gray 3")!
+      }
+
+      static var gray4: UIColor {
+        UIColor(named: "Gray 4")!
+      }
```

Comment:
```suggestion
      static var black: UIColor {
        UIColor(named: "Black")!
      }

      static var gray1: UIColor {
        UIColor(named: "Gray 1")!
      }

      static var gray2: UIColor {
        UIColor(named: "Gray 2")!
      }

      static var gray3: UIColor {
        UIColor(named: "Gray 3")!
      }

      static var gray4: UIColor {
        UIColor(named: "Gray 4")!
      }
```

Let's update black to have the same color incremental values. ex. 20-40-60-80-100(Pure Black)

Reply:
Just to be sure, Should we add shades to the black from black 20 to black 100? and should I also do this to the rest of the colors?

Reply:
Oh so we change the grays to black 20-80 and the current black to Black 100?

Reply:
Yes exactly, just the black.